### PR TITLE
GLSTMCell fix

### PR DESCRIPTION
**Pull request by okuchaiev**
* Make sure it works correctly for input_size != num_units

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 107 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 122 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 137 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 152 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 167 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 182 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 197 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

File: tensorflow/contrib/rnn/python/ops/rnn_cell.py [hunk 1]
```diff
@@ -2354,9 +2355,9 @@ def call(self, inputs, state):
           x_g_id = array_ops.concat(
               [
                   self._get_input_for_group(inputs, group_id,
-                                            self._group_shape[0]),
+                                            int(input_size / self._number_of_groups)),
```
Comment (ebrevdo) [inline tensorflow/contrib/rnn/python/ops/rnn_cell.py]: this won't work if input_size is a tensor.

File: tensorflow/contrib/rnn/python/ops/rnn_cell.py [hunk 2]
```diff
@@ -2354,9 +2355,9 @@ def call(self, inputs, state):
...
                   self._get_input_for_group(m_prev, group_id,
-                                            self._group_shape[0])
+                                            int(self._output_size / self._number_of_groups))
```
Comment (ebrevdo) [inline tensorflow/contrib/rnn/python/ops/rnn_cell.py]: ditto

**Review by ebrevdo — CHANGES_REQUESTED** [review]
can you add a unit test?

**Comment by ebrevdo** [comment]
@qlzh727 is taking lead on deciding which cells in tf.contrib.rnn to keep.  Scott - PTAL?

**Comment by qlzh727** [comment]
Seems that there is a merge conflict with the existing file, can u rebase to the latest?

Agree with ebrevdo@ that a unit test is need to demonstrate that the change fixed the specific problem, which will prevent the problem happening from any future change.

**Comment by okuchaiev** [comment]
@ebrevdo and @qlzh727 thanks for looking into this!
I've reviewed merge conflicts and apparently someone else has fixed this bug. 
The master looks good. Closing, because nothing needs to change in the code.

**Pull request closed without merging**